main.tf

```diff
@@ -121,7 +121,7 @@ resource "aws_subnet" "lambda" {
 }
 
 resource "aws_route_table" "public" {
-  count  = min(local.public_subnets, 1)
+  count  = var.additional_route_tables ? local.public_subnets : min(local.public_subnets, 1)
```

Comment:
This will need to be inverted as now the default is `true` instead of `false`.

Reply:
again why ?

Reply:
as above.

Reply:
updated to new variables and tested